storage/storage-interface/src/async_proof_fetcher.rs

```diff
@@ -4,7 +4,7 @@
 use crate::{proof_fetcher::ProofFetcher, DbReader};
 
 use crate::metrics::TIMER;
-use anyhow::Result;
+use anyhow::{format_err, Result};
```

Comment:
nit: I like anyhow! better because it's shorter.. we are putting in more format_err!s mostly in files that are already using format_err! if I'm not mistaken. but your choice..

Reply:
done